code/addons/interactions/template/stories/basics.stories.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jest/no-standalone-expect */
+import globalThis from 'global';
+import { within, waitFor, fireEvent, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+export default {
+  component: globalThis.Components.Form,
+  argTypes: {
+    onSuccess: { type: 'function' },
+  },
+};
+
+export const Type = {
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await userEvent.type(canvas.getByTestId('value'), 'test');
+  },
+};
+
+export const Step = {
+  play: async (context) => {
+    await context.step('Enter value', async () => Type.play(context));
+  },
```

Comment:
```suggestion
  play: async ({ step }) => {
    await step('Enter value', Type.play);
  },
```
I think you can just do that. Should verify though.